dotCMS/src/integration-test/java/com/dotcms/vanityurl/business/VanityUrlAPITest.java

```diff
@@ -1533,31 +829,23 @@ public void Test_Vanity_URI_Ending_With_Forward_Slash_Handles_Non_forward_Slash_
         final int order = 1;
 
         final Contentlet vanityURL = filtersUtil.createVanityUrl(title, hostIdentifier, uri,
-                forwardTo, action, order, defaultLanguageId);
+                forwardTo, action, order, defaultLanguage.getId());
         Assert.assertNotNull(vanityURL);
         filtersUtil.publishVanityUrl(vanityURL);
 
         final String testUri1 = "/test1_" + timeMillis + "/" ;
         final String testUri2 = "/test1_" + timeMillis  ;
 
-        final CachedVanityUrl v1 = vanityUrlAPI.getLiveCachedVanityUrl(testUri1, host, defaultLanguageId, APILocator.systemUser());
+        final CachedVanityUrl v1 = vanityUrlAPI.resolveVanityUrl(testUri1, host, defaultLanguage).get();
         Assert.assertNotNull(v1);
-        Assert.assertNotNull(v1.getVanityUrlId());
-        Assert.assertNotEquals(VanityUrlAPI.CACHE_404_VANITY_URL,v1.getVanityUrlId());
-        Assert.assertEquals(v1.getForwardTo(),forwardTo);
+        Assert.assertNotNull(v1.vanityUrlId);
+        Assert.assertEquals(v1.forwardTo,forwardTo);
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'Assert.assertEquals' due to existing static import 'org.junit.Assert.assertEquals'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5434315)